### ISSUE TITLE
Liquidation Button - Enhancement

### ISSUE THREAD

**Issue by jjc6676** [cryptoeax/arbbot#138]
I was thinking about this earlier today and we can leave this up for debate of course but hear me out.

I think it would be a nice enhancement to be able to **_liquidate_** all of your coins with a single button or command. I am not sure of the implementation just yet, but I assume some of us would like to take our bot's BTC and put it all into one coin at some point.

In other words, I think an "**auto-liquidate**" feature would be nice to have or could be nice to have in the future... especially if we want to cash out back to BTC if we believe it will go to 20k or more again.

**Comment by cryptoeax** [comment]
Yes, I have been thinking of adding some controls for this at some point.  Note that a lot of details need to be figured out on how to implement this, since in general having bots place market orders is very unsafe, and I would be hesitant to do that.  For example, as you may have noticed, the current conservative algorithm the bot uses in the liquidation module sometimes takes weeks to liquidate an asset...  :-)
